<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<<<<<<< HEAD
<testsuite ignored="0" hostname="mm.home" failures="0" tests="2" name="Default test" time="2.17" errors="0" timestamp="17 Apr 2021 18:08:26 GMT">
  <testcase classname="jl.project._chrometests.UserRequirement1_Test" name="@BeforeMethod navigate" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat jl.project._chrometests.UserRequirement1_Test.navigate(UserRequirement1_Test.java:59)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.base/java.lang.reflect.Method.invoke(Method.java:566)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeMethod(Invoker.java:516)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- @BeforeMethod navigate -->
  <testcase classname="jl.project._chrometests.UserRequirement1_Test" name="@BeforeMethod navigate" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod navigate -->
  <testcase classname="jl.project._chrometests.UserRequirement1_Test" name="@AfterClass releaseResources" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterClass releaseResources -->
  <testcase classname="jl.project._chrometests.UserRequirement1_Test" name="createCategory" time="0.0">
    <skipped/>
  </testcase> <!-- createCategory -->
  <testcase classname="jl.project._chrometests.UserRequirement1_Test" name="deleteCategory" time="0.0">
    <skipped/>
  </testcase> <!-- deleteCategory -->
=======
<testsuite ignored="0" hostname="jl-Ubuntu" failures="0" tests="1" name="Default test" time="108.769" errors="0" timestamp="16 Apr 2021 15:35:09 GMT">
  <testcase classname="jl.project.pa11ytest.UserRequirement5_Pa11yTest" name="pa11yTest" time="0.011"/>
>>>>>>> 28d7f2d8272f88ade4174193ce9d17c8d0a62b6d
</testsuite> <!-- Default test -->
